src/Sulu/Bundle/PreviewBundle/Tests/Unit/Preview/Object/PreviewObjectProviderRegistryTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PreviewBundle\Tests\Unit\Preview\Object;
+
+use PHPUnit\Framework\TestCase;
+use Sulu\Bundle\PreviewBundle\Preview\Exception\ProviderNotFoundException;
+use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderInterface;
+use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderRegistry;
+use Sulu\Bundle\PreviewBundle\Preview\Object\PreviewObjectProviderRegistryInterface;
+
+class PreviewObjectProviderRegistryTest extends TestCase
+{
+    /**
+     * @var PreviewObjectProviderRegistryInterface
+     */
+    private $objectProviderRegistry;
+
+    /**
+     * @var PreviewObjectProviderInterface
+     */
+    private $provider;
+
+    /**
+     * @var string
+     */
+    private $providerKey = 'test-provider';
```

Comment:
`previewObjectProviderKey`, resp. wouldn't it make more sense to not use this variable?